apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/api/common/logging/LoggingFilterTest.java

```diff
@@ -0,0 +1,504 @@
+package io.stargate.sgv2.api.common.logging;
+
+import static io.stargate.sgv2.api.common.config.constants.LoggingConstants.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.stargate.sgv2.api.common.StargateRequestInfo;
+import io.stargate.sgv2.api.common.config.LoggingConfig;
+import jakarta.ws.rs.container.ContainerRequestContext;
+import jakarta.ws.rs.container.ContainerResponseContext;
+import java.io.ByteArrayInputStream;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.shaded.org.bouncycastle.util.Strings;
+
+public class LoggingFilterTest {
+  private static final String TEST_TENANT = "test-tenant";
+  private static final String TEST_TENANT_2 = "test-tenant-2";
+
+  @Test
+  public void testIsLoggingNotAllowedWithDefaults()
+      throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+    ContainerRequestContext requestContext = mock(ContainerRequestContext.class);
+    ContainerResponseContext responseContext = mock(ContainerResponseContext.class);
+    LoggingConfig loggingConfig = mock(LoggingConfig.class);
+    StargateRequestInfo stargateRequestInfo = mock(StargateRequestInfo.class);
+    mockLoggingConfig(loggingConfig, false, null, null, null, null, null);
+    mockStargateRequestInfo(stargateRequestInfo);
+    when(requestContext.getUriInfo()).thenReturn(mock(jakarta.ws.rs.core.UriInfo.class));
+    when(requestContext.getUriInfo().getPath()).thenReturn("/api/rest/keyspaces");
+    when(requestContext.getMethod()).thenReturn("GET");
+    when(responseContext.getStatus()).thenReturn(200);
+
+    LoggingFilter loggingFilter =
+        new LoggingFilter(new ObjectMapper(), stargateRequestInfo, loggingConfig);
+    Method isAllowedMethod =
+        loggingFilter
+            .getClass()
+            .getDeclaredMethod(
+                "isLoggingAllowed", ContainerRequestContext.class, ContainerResponseContext.class);
+    isAllowedMethod.setAccessible(true);
+    boolean isAllowed =
+        (boolean) isAllowedMethod.invoke(loggingFilter, requestContext, responseContext);
+    assert !isAllowed;
```

Comment:
Should use assert method from junit instead of Java language assert keyword?

Reply:
Fixed in [06db8de](https://github.com/stargate/stargate/pull/2754/commits/06db8de3d2aa336086250226249c76497bcbe1e2)